llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -464,6 +464,19 @@ them.
  Read a list of symbols from <filename> and change their visibility to the
  specified value. Visibility values: default, internal, hidden, protected.
 
+.. option:: --skip-symbol <symbol>
+
+ Do not change the parameters of symbol ``<symbol>`` when executing other
```

Comment:
Should this mention that this does not prevent removing symbols?

Reply:
I don't think it needs to: it explicitly states the things that it impacts (name/binding/visibility). It's probably harmless to add it though.